nilearn/datasets/func.py

```diff
@@ -1894,45 +1957,66 @@ def infer_subjects(urls):
 
 
 def patch_openneuro_dataset(file_list):
-    """Add symlinks for files not named according to latest :term:`BIDS` conventions.
+    """Add symlinks for files not named according to :term:`BIDS` conventions.
+
+    This function uses a series of hardcoded patterns to generate the corrected
```

Comment:
Maybe we can put this in a warning block?

Reply:
Done!